pkg/ghclients/ghclients.go

```diff
@@ -45,6 +46,7 @@ type GHClients struct {
 	clients map[int64]*github.Client
 	tr      http.RoundTripper
 	key     []byte
+	cache   *memoryCache
```

Comment:
Nit: use `httpcache.Cache` instead of `*memoryCache`

Reply:
As mentioned, I need to have the literal type to either access Items or a log fn.